config/initializers/00_dotenv.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
```

Comment:
Do these configs should be in a separate file? or Should they live in development, staging, production rb files?

Reply:
This config is build based on .env. so, I don't think we need separeted files, only we need to have the correct env variables in each environment